data/Younus-Saberi.json

```diff
@@ -26,24 +26,46 @@
   ],
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "group":"Personal",
+      "name": "My Portfolio Website 🧑‍💻",
+      "url": "https://younussaberi.me/PersonalWebsite/",
+      "icon": "FaLink"
+    },
+    {
+      "group":"Personal",
+      "name": "Resume 📄",
+      "url": "https://drive.google.com/file/d/1TLVk4vwgDLubI8zEzH9PHNYNzBotX4a0/view?usp=share_link",
+      "icon": "FaBookmark"
+    },
+    {
+      "group":"Socials",
+      "name": "Twitter: Follow me on Twitter 🐦",
       "url": "https://twitter.com/younussaberi",
       "icon": "FaTwitter"
     },
     {
-      "name": "Let's connect on Linkedin 🤝 ",
+      "group":"Socials",
+      "name": "Linkedin: Let's connect on Linkedin 🤝 ",
       "url": "https://twitter.com/younussaberi",
```

Comment:
You may want to use your LinkedIn url here :+1: 
Rest everything looks good :tada: